Gems/Atom/RHI/Vulkan/3rdParty/Platform/Linux/glad_vulkan_linux.cmake

```diff
@@ -6,6 +6,18 @@
 #
 #
 
-set(GLAD_VULKAN_COMPILE_DEFINITIONS 
-    VK_USE_PLATFORM_XCB_KHR
-)
+if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
+    set(GLAD_VULKAN_COMPILE_DEFINITIONS 
+        VK_USE_PLATFORM_XCB_KHR
+    )
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "wayland")
+    set(GLAD_VULKAN_COMPILE_DEFINITIONS 
+        VK_USE_PLATFORM_WAYLAND_KHR
+    )
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "xlib")
+    set(GLAD_VULKAN_COMPILE_DEFINITIONS 
+        VK_USE_PLATFORM_XLIB_KHR
+    )
+else()
+    message(FATAL_ERROR, "Linux Window Manager ${PAL_TRAIT_LINUX_WINDOW_MANAGER} is not recognized")
+endif()
```

Comment:
repeated nitpick: Can you add a newline to this file

Reply:
Done